### PR TITLE
Replace Red Cross with something actually useful

### PR DESCRIPTION
**Pull request by Pernat1y**
Removed Red Cross donation link, as they spending money in a terrible way.
Sources:
https://mobile.twitter.com/your_psychoo/status/1507110118906863617?s=21&t=8Wz67Xfp19a3lT6r0VaIwg
https://life.pravda.com.ua/society/2022/03/26/247987/
https://telegraf.com.ua/ukr/ukraina/2022-03-25/5700516-na-krasnom-kreste-mozhno-stavit-krest-ukraintsev-vozmutili-strannye-dogovorennosti-s-lavrovym
https://nv.ua/ukr/ukraine/events/chervoniy-hrest-skandal-z-ofisom-u-rostovi-nacionalniy-komitet-poyasniv-situaciyu-novini-ukrajini-50228742.html

### CODE REVIEW

**Comment by garritfra** [comment]
Thanks for the heads up. The website you proposed doesn't seem to be available in english, but they claim to have a new site which is:
https://www.comebackalive.in.ua/

Do you have any insights if this is a viable alternative?

**Comment by Pernat1y** [comment]
They have English version - https://savelife.in.ua/en/donate/
https://www.comebackalive.in.ua/ will work too (it is, basically, same website).

Another option is
https://bank.gov.ua/en/news/all/natsionalniy-bank-vidkriv-spetsrahunok-dlya-zboru-koshtiv-na-potrebi-armiyi

File: components/Header.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ export default function Header(props) {
     return (
         <header className="header">
             <a
-                href="https://donate.redcrossredcrescent.org/ua/donate/~my-donation?_cv=1"
+                href="https://savelife.in.ua/donate/"
```
Comment (garritfra) [inline components/Header.js:5]:
```suggestion
                href="https://www.comebackalive.in.ua/"
```

**Comment by garritfra** [comment]
Thanks again, and all the best to you!

**Pull request merged**